doc/README.modern_cpp.md

```diff
@@ -1,11 +1,12 @@
 # Modern C++ features
 
-We define here as "modern C++" features the ones introduced since C++17. These features should be used in a "controlled" form in Firebird code, as not all compilers fully support them.
+It's allowed to use in Firebird internal code C++ features up to and including C++17.
 
-Only ones mentioned in this document could be used, but as necessities appears, discussion should be started in the devel list or pull requests.
+Public files (as API headers) is limited to C++11.
```

Comment:
Note that I also want to change version of public code to C++11.

Reply:
How this limitation can be enforced on practice ?

Reply:
We may build the examples with this option.

Reply:
So now API headers can handle std::exception?

Reply:
`std::exception` is pre-c++11 feature, so explain what you want in fb-devel, as it seems to have nothing related with this discussion.